app/components/AppSettingsAccountAddView.vue

```diff
@@ -60,6 +69,9 @@ export default Vue.extend({
                 this.busy = false;
             }
         },
+        goBack(ev) {
```

Comment:
I'm trying to ensure we add types to function returns and parameters from now on. Not a big deal here but just something to keep in mind.